pkg/performanceprofile/controller/performanceprofile_controller.go

```diff
@@ -201,6 +227,40 @@ func (r *PerformanceProfileReconciler) tunedProfileToPerformanceProfile(tunedPro
 	return requests
 }
 
+func (r *PerformanceProfileReconciler) ctrRuntimeConfToPerformanceProfile(ctrRuntimeConfObj client.Object) []reconcile.Request {
+	ctrcfg := &mcov1.ContainerRuntimeConfig{}
+
+	err := r.Get(context.TODO(), client.ObjectKeyFromObject(ctrRuntimeConfObj), ctrcfg)
```

Comment:
*NOT FOR THIS PR*: in the future we need to stop hardcoding `context.TODO()` and properly pass down the call chain in the reconcile loop the given context.